### ISSUE TITLE
`silu_out` op lowering

### ISSUE THREAD

**Issue by tmabraham** [pytorch/xla#2717]
## 🚀 Feature

Lowering of `silu_out`

## Motivation

I want to use EfficientNet models as described [here](https://github.com/pytorch/xla/issues/2715).

## Pitch

Implementation of `silu_out` op lowering.

## Alternatives

I am not sure if there are any?

## Additional context

Issue #2715

**Comment by JackCaoG** [comment]
@tyoc213 Do you want to give this one a shot? We have `sigmoid` and `log_sigmoid` lowered already. You can find it in 
https://github.com/pytorch/xla/blob/3aebd8adf46261bf548e4597c19ffd2577d3d39c/torch_xla/csrc/aten_xla_type.cpp#L1836

According to https://pytorch.org/docs/stable/generated/torch.nn.SiLU.html, `silu` should be pretty similar

**Comment by tyoc213** [comment]
yeah @JackCaoG I will give a shot at it, first Q before check the ref: will need to modify `gen.py` for this one (because I have seen some ops modify it and others that not but not sure which ones)

**Comment by JackCaoG** [comment]
@tyoc213 You do not need to modify the `gen.py` for this one. The point of adding op to `_FN_OUT` in `gen.py` is if we provide the lowering for an op (for example `abs`), adding `op_out` to `_FN_OUT` will generate the `op_out`(`abs_out` in this case) that uses our lowering of the op. 

In this case, there isn't an op named `silu`(you can check`aten_xla_type_default.h`), so you can directly lower the `silu_out`
